### PR TITLE
Component: Specification Block

### PR DESCRIPTION
**Pull request by bsmrdel101**
### Description:

This component will display a block of data representing Focus Expectations, Keyboard Interactions, Semantic Elements, and ARIA Roles and Attributes.

Screen Shot 2022-10-20 at 12 08 20 PM

### Spec:

Designs: [Figma](https://www.figma.com/file/UjMQEwUvAsnFAMhYNdUiVT/Apprenticeship-Capstone?node-id=84%3A663)

See Story: [FSA22V2-259](https://sparkbox.atlassian.net/jira/software/c/projects/FSA22V2/boards/125?modal=detail&selectedIssue=FSA22V2-259)

### Validation:

- [x] This PR has code changes, and our linters still pass.
- [x] This PR has new code, so new tests were added or updated, and they pass.

#### To Validate:

1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, etc).
2. Pull down all related branches.
3. In terminal: `npm run dev`.
4. Run `npm run lint`.
5. Run axe Devtools on affected pages.
6. Confirm all tests pass.
7. Go to [localhost:3000](http://localhost:3000/accordion) to see the details page, for the accordion component. You should see the 8. specification block components there in a grid layout. This should also show up for other components detail pages (/dialog, /disclosure, /tabs).
9. Verify that grid is responsive.

### CODE REVIEW

**Review by tisidro — DISMISSED** [review]
This appears to match the basic styling (more involved styling appears to be coming later on [in this card](https://sparkbox.atlassian.net/browse/FSA22V2-243?atlOrigin=eyJpIjoiYzBhOTcyNWFkM2EyNDczN2JlZmM3MjFkZTEwY2NiODUiLCJwIjoiaiJ9).  I did get a weird message in AxeDevtools, but that's probably going away once we add in the images and so forth. Or--those might be an issue with contrast. We will see!
image

Otherwise, I don't see any issues. Good work Bennett--you are really cranking these out! :)

**Review by tisidro — COMMENTED** [review]
One thing I do want to point out-- when I went into your test file and deleted paragraph text, the tests still passed. Should it have failed based on this?

image

File: src/styles/components/_detailsPage.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+$breakpoint-tablet: 820px;
+
+.cmp-specifications-list {
+  display: block;
+
+  @media only screen and (min-width: $breakpoint-tablet) {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+}
```
Comment (mlogsdon18) [inline src/styles/components/_detailsPage.scss:10]: Is there a `detailsPage` component? Would it be better to name this file `_specificationBlock.scss`?

**Review by mlogsdon18 — APPROVED** [review]
Left one comment about a file name but I will go ahead and approve since I don't know all the project details. Looks great!

**Review by tisidro — APPROVED** [review]
Looks good Bennett! I still wasn't able to get a failing test, but I guess this could be refactored later if needed to take in dynamic values (?). We should ask about that during standup.

**Review by tisidro — APPROVED** [review]

**Pull request merged**